src/main/java/com/dawn/jat/illuminati/post/advice/PostExceptionAdvice.java

```diff
@@ -1,15 +1,31 @@
 package com.dawn.jat.illuminati.post.advice;
 
+import com.dawn.jat.illuminati.post.exception.ErrorResponse;
 import com.dawn.jat.illuminati.post.exception.PostNotFoundException;
+import com.dawn.jat.illuminati.post.exception.PostSummaryNotFoundException;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 
 @ControllerAdvice
 public class PostExceptionAdvice {
-    @ExceptionHandler(value = PostNotFoundException.class)
+    /**
+     * Throw Exception when the query Slug Unavailable.
+     */
+    @ExceptionHandler(PostNotFoundException.class)
     public ResponseEntity<Object> exception(PostNotFoundException exception) {
         return new ResponseEntity<>(exception.getMessage(), HttpStatus.NOT_FOUND);
     }
+
+    /**
+     * Throw Exception if Post Summary has No Content.
+     */
+    @ExceptionHandler(PostSummaryNotFoundException.class)
+    public ResponseEntity<ErrorResponse> exceptionPostSummary(Exception exception) {
+        ErrorResponse error = new ErrorResponse();
+        error.setErrorCode(HttpStatus.NO_CONTENT.toString());
```

Comment:
Fix the response to this  to keep it the same with the success one
```
{
    error: {
        code: "204",
        message: "Cannot...."    
    },
    success: false
}
```

Reply:
Updated